ledger/catchpointtracker.go

```diff
@@ -346,7 +354,9 @@ func (ct *catchpointTracker) loadFromDisk(l ledgerForTracker, dbRound basics.Rou
 		return err
 	}
 
+	ct.cachedDBRound = dbRound
```

Comment:
Sanity check: shouldn't this be protected by `catchpointsMu`?

Reply:
yes, it should. I'll update.